### DISCUSSION TITLE
input in flex scanner failed

### DISCUSSION THREAD

**Discussion by dllu** [davatorium/rofi#1865] [(Possible) Issues]
I'm running into a issue with `input in flex scanner failed`. When searching for the issue I found it's mostly because people didn't point it towards the right config path (for other projects and not rofi in particular). I'm probably missing something silly but I can't see how I could have possibly made a mistake here (I've also successfully installed rofi and used it with the exact same config on 3 other machines with no issues).

~~Config~~ Theme: https://github.com/dllu/dotfiles/blob/main/rofi.rasi

To reproduce:

* On a clean install of Ubuntu 20.04, install rofi with `sudo apt install rofi` and then move the linked `rofi.rasi` to `~/.config/rofi/dllu.rasi`.
* Run `rofi -show run -theme dllu -font "monospace 9"`

```
dllu$ rofi -show run -theme dllu -font "monospace 9"
input in flex scanner failed
dllu$ echo $?
2
dllu$ rofi -v
1.5.4
dllu$ ls -al ~/.config/rofi/dllu.rasi 
-rw-r--r-- 1 dllu dllu 3868 Jul  3 18:31 /home/dllu/.config/rofi/dllu.rasi
dllu$ echo $XDG_CONFIG_DIR

```

However, just `rofi -show run` by itself works without the theme.

**Comment by DaveDavenport** [comment]
The version of rofi used is 4 years old. This is no longer supported.

You did not paste the theme (dllu.rasi) that fails, I had to read a magic setup.sh to figure out that what you linked as config is the theme?

Your 'config' aka 'theme' work fine for me on latest rofi.

**Reply by DaveDavenport** [reply]
The linked theme also works on 1.5.4.
Do you have a config for rofi?

Can you try on latest git? (should compile fine on 20.04) then I could add some extra debug if needed.

**Reply by dllu** [reply]
I don't have a config besides that theme. Sorry about the confusion regarding config vs theme and the filename! >__<"

And yes, I have used the same theme on several different systems and it works every time, except on this current ubuntu 20.04 machine that I'm using.

**Reply by DaveDavenport** [reply]
I've added an extra debug to that error message in git.
It seems to happen when a read (of the theme file) gives an error.
